app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -239,7 +240,7 @@ class ExplorationActivityTest {
     ).use {
       explorationDataController.startPlayingExploration(TEST_EXPLORATION_ID_2)
       openActionBarOverflowOrOptionsMenu(context)
-      onView(withText(context.getString(R.string.help))).perform(click())
+      onView(withText(context.getString(R.string.help_activity_title))).perform(click())
```

Comment:
Use `help_menu` here as it is part of menu.

Reply:
changed help_activity_title to menu_help